lab-sarah/ng-template/app/component/gallery/edit-gallery/edit-gallery.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  template: require('./edit-gallery.html'),
+  controller: ['$log', 'galleryService',  EditGalleryController],
+  controllerAs: 'editGalleryCtrl',
+  bindings: {
+    gallery: '<',
+  },
+};
+
+function EditGalleryController($log, galleryService){
+  $log.debug('init editGalleryCtrl');
+
+  this.updateGallery = function(){
+    galleryService.updateGallery(this.gallery, this.gallery._id);
```

Comment:
Good job utilizing your service and making use of the gallery property added to the scope of your controller through the use of bindings!